test/fixtures/cookbooks/test/metadata.rb

```diff
@@ -1,13 +1,13 @@
-maintainer "Lamont Granquist"
-maintainer_email "lamont@scriptkiddie.org"
-license "Apache 2.0"
-source_url "https://github.com/chef-cookbooks/habitat"
-issues_url "https://github.com/chef-cookbooks/habitat/issues"
-description "test"
-long_description "test"
-version "0.0.1"
-name "test"
+maintainer 'Lamont Granquist'
+maintainer_email 'lamont@scriptkiddie.org'
+license 'Apache 2.0'
+source_url 'https://github.com/chef-cookbooks/habitat'
+issues_url 'https://github.com/chef-cookbooks/habitat/issues'
+description 'test'
+long_description 'test'
+version '0.0.1'
+name 'test'
 
-depends "habitat"
+depends 'habitat'
 
-chef_version ">= 12.0.3" if respond_to?(:chef_version)
+chef_version '>= 12.0.3' if respond_to?(:chef_version)
```

Comment:
All this extra meta data should really come out of the test cookbook. chef_version isn't even right.